### ISSUE TITLE
[Android] displayIncomingCall inoperative 

### ISSUE THREAD

**Issue by dkahdwk** [react-native-webrtc/react-native-callkeep#717]
# Bug report

- [x] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [x] Android
 - [ ] iOS

## Description

- When executing a`displayIncomingCall`, no action is performed without any errors.
- I found a strange bug, and it just works when I call anyone after running a `displayIncomingCall`
  - ![KakaoTalk_Photo_2023-07-17-18-26-17 (1)](https://github.com/react-native-webrtc/react-native-callkeep/assets/56759331/8eeb38a5-bb87-456c-8a27-b2e595500428)
  - CallKeep only works when you call any number like this (It doesn't print out on the app.)

## Steps to Reproduce

build.gradle
```
  buildToolsVersion = "33.0.0"
  minSdkVersion = 21
  compileSdkVersion = 33
  targetSdkVersion = 33
```

callkeep
```jsx

  const didReceiveStartCallAction = ({ handle, callUUID }: {
    handle: string;
    callUUID?: string | undefined;
  }) => {
    if (!callUUID) return;
    RNCallKeep.startCall(callUUID, handle, handle);
  };

  const displayIncomingCall = ({ localCallUUID, callNumber, caller }: DisplayIncomingCallTypes) => {

    /**
     * localUUID = '0260855b-17c1-457a-99db-013181dd85b5'
     * callNumber = '010-1234-5678'
     * caller = 'person'
     */
    RNCallKeep.displayIncomingCall(localCallUUID, callNumber, caller, 'number', true);
  };

  ...
  useEffect(() => {
    RNCallKeep.addEventListener('didReceiveStartCallAction', didReceiveStartCallAction);

    return () => {
      RNCallKeep.removeEventListener('didReceiveStartCallAction');
    };
  }, []);

```

## Versions
    - Callkeep: 4.3.9
    - React Native: 0.71.12
    - iOS:
    - Android: 13 (One UI 5.1)
    - Phone model: galaxy s21

## Logs

```
Paste here
```

**Comment by MARK-46** [comment]
same bug :(
```
- Callkeep: 4.3.9
- React Native: 0.72.1
- Not Working
   - Android: 13 (Realme UI 4.0)
   - Phone model: Realme GT Master Edition
- Working
   - Android: 12
   - Phone model: Xiaomi Redmi Note 5 Pro
```
```gradle
buildToolsVersion = "33.0.0"
minSdkVersion = 21
compileSdkVersion = 33
targetSdkVersion = 33
```
```js
import RNCallKeep from 'react-native-callkeep';

const options = {
  android: {
    alertTitle: 'Permissions Required',
    alertDescription:
      'This application needs to access your phone calling accounts to make calls',
    cancelButton: 'Cancel',
    okButton: 'ok',

    selfManaged: true, // true - no errors
                       // false - errors : registerPhoneAccount *** 
                       //   java.lang.IllegalArgumentException: Error, cannot change a self-managed phone account ComponentInfo{com.ats/io.wazo.callkeep.VoiceConnectionService}, ***, UserHandle{0} to other kinds of phone account
  },
};

RNCallKeep.setup(options).then(state => {
  console.log('state: ', state); // output log - false
  RNCallKeep.setAvailable(true);
  RNCallKeep.displayIncomingCall(
    '467d2d89-897c-477d-b07e-27351f1003cd',
    '123',
    'shb',
    'number',
    false,
    {
      supportsDTMF: true,
      supportsHolding: true,
    },
  );
});
```

**Comment by dkahdwk** [comment]
@MARK-46 
Found a cause for this!
 (As a result of configuring and testing the new project, the bug did not occur, so I found it during the test by comparing it.)

1. `AndroidManifest.xml`
remove `<uses-permission android:name="android.permission.READ_CALL_LOG"/>`
remove `<uses-permission android:name="android.permission.MANAGE_OWN_CALLS"/>`
remove `<uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>`

2. `RNCallKeep.setup`
remove `selfManaged` (If you removed the permission for item 1, clearing `selfManaged` will not cause an error!)

3. `displayIncomingCall` success after rebuild! 🥹
